pkg/releaser/releaser.go

```diff
@@ -144,15 +162,62 @@ func (r *Releaser) UpdateIndexFile() (bool, error) {
 		}
 	}
 
-	if update {
-		fmt.Printf("--> Updating index %s\n", r.config.IndexPath)
-		indexFile.SortEntries()
-		return true, indexFile.WriteFile(r.config.IndexPath, 0644)
-	} else {
-		fmt.Printf("--> Index %s did not change\n", r.config.IndexPath)
+	if !update {
+		fmt.Printf("Index %s did not change\n", r.config.IndexPath)
+		return false, nil
+	}
+
+	fmt.Printf("Updating index %s\n", r.config.IndexPath)
+	indexFile.SortEntries()
+
+	if err := indexFile.WriteFile(r.config.IndexPath, 0644); err != nil {
+		return false, err
+	}
+
+	if !r.config.Push && !r.config.PR {
+		return true, nil
+	}
+
+	worktree, err := r.git.AddWorktree("", "origin/"+r.config.PagesBranch)
```

Comment:
Don't want to scope creep too hard here, but we may now want to add a `remote` config option as well?

Reply:
Good point. Added it.